### PR TITLE
Two new functions

### PR DESCRIPTION
**Pull request by nbrl**
Yesterday, it was decided that we need some new external functions, getShareToken and sendSharepointData (for now). This is their implementations - they are subtle wrappers around sendSharepoint.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "laxbreak": true            // allow multi-line ternary operator
```
Comment (jabclab) [inline .jshintrc]: Interesting that GitHub seems to think that `.jshintrc` is a JSON file :smile:

Comment (nbrl) [inline .jshintrc]: *shrugs* You'd think either JS or RC...

File: src/sdk.js [hunk 2]
```diff
@@ -377,6 +379,26 @@
...
+        }
+
+        var dataPrep = AT._prepareData(data || window.advocate_things_data); // using for clone
+
+        // Remove dataPrep._at.sharepointName if it is present
+        if (dataPrep && dataPrep._at) {
+            delete dataPrep._at.sharepointName;
+        }
+
+        return AT.sendSharepoint(null, dataPrep, function (err, res) {
```
Comment (jabclab) [inline src/sdk.js]: Do we want to handle the case where `err` is set?

**Comment by jabclab** [comment]
Seem to be missing JSDoc for the new methods.

**Comment by jabclab** [comment]
New methods not referenced in `README.md`.

File: src/sdk.js (hunk 2 above)
Comment (nbrl) [inline src/sdk.js]: Done.

**Comment by nbrl** [comment]
We are now exposing some semantically-improved functions, so these are no longer required.

**Pull request closed without merging**